x-pack/plugins/cases/public/containers/use_delete_comment.test.tsx

```diff
@@ -5,94 +5,114 @@
  * 2.0.
  */
 
+import React from 'react';
 import { act, renderHook } from '@testing-library/react-hooks';
 import type { UseDeleteComment } from './use_delete_comment';
 import { useDeleteComment } from './use_delete_comment';
 import * as api from './api';
 import { basicCaseId } from './mock';
+import { TestProviders } from '../common/mock';
 import { useRefreshCaseViewPage } from '../components/case_view/use_on_refresh_case_view_page';
+import { useToasts } from '../common/lib/kibana';
 
 jest.mock('../common/lib/kibana');
 jest.mock('./api');
 jest.mock('../components/case_view/use_on_refresh_case_view_page');
 
 const commentId = 'ab124';
 
+const wrapper: React.FC<string> = ({ children }) => <TestProviders>{children}</TestProviders>;
+
 describe('useDeleteComment', () => {
+  const addSuccess = jest.fn();
+  const addError = jest.fn();
+
+  (useToasts as jest.Mock).mockReturnValue({ addSuccess, addError });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
   it('init', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseDeleteComment>(() =>
-        useDeleteComment()
-      );
-      await waitForNextUpdate();
-      expect(result.current).toEqual({
-        isError: false,
-        deleteComment: result.current.deleteComment,
-      });
-    });
+    const { waitForNextUpdate, result } = renderHook<string, UseDeleteComment>(
+      () => useDeleteComment(),
+      {
+        wrapper,
+      }
+    );
+
+    expect(waitForNextUpdate).toBeTruthy();
```

Comment:
I think we should `await` not `expect` the `waitForNextUpdate`, correct?

Reply:
yes